cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
Something goes completely wrong with your platform. With OpenMotes, I do not see any `NOACK`s at such a distance. In particular, what should not be shown is `FIFOP unexpected`.